pkg/release/release.go

```diff
@@ -33,3 +42,30 @@ var Image = map[string]string{
 	"pod":                       "quay.io/microshift/pause:" + Base,
 	"service_ca_operator":       "quay.io/microshift/service-ca-operator:" + Base,
 }
+
+func init() {
+	arch_replacer := strings.NewReplacer("amd64", "x86_64", "arm64", "aarch64")
+	arch := arch_replacer.Replace(runtime.GOARCH)
+
+	release_file := "release/release-" + arch + ".json"
```

Comment:
It would be great if we could also produce a toml snippet so customers just can append that to their blueprints without additional processing if they wanted.

I suggest that as follow up, not now.